internal/kibana/policies.go

```diff
@@ -52,7 +52,7 @@ func (c *Client) CreatePolicy(p Policy) (*Policy, error) {
 func (c *Client) DeletePolicy(p Policy) error {
 	reqBody := `{ "agentPolicyId": "` + p.ID + `" }`
 
-	statusCode, respBody, err := c.post("agent_policies/delete", []byte(reqBody))
+	statusCode, respBody, err := c.post(fmt.Sprintf("%s/agent_policies/delete", FleetAPI), []byte(reqBody))
```

Comment:
nit: Hm.. I'm wondering if it's not better to extract also the resource part (agent_policies, package_policies) as they appear in multiple places. Once it's changed, I think you can switch from `fmt.Sprintf` to a simple concatenation (URL prefix + operation).

Reply:
I think it's more readable this way, actually. And while the prefixes are more likely to change (due to project name changes, etc.), I think the resource paths are less likely to change. So I would prefer to keep it as-is.